sql_grader/scenarios/SQL-Labs.xml

```diff
@@ -0,0 +1,266 @@
+<sequence_demo>
+    <vertical_demo>
+        <sql_grader
+            dataset="rating"
+            display_name="Q1"
+            answer_query="SELECT title FROM Movie where Director = 'Steven Spielberg';"
+            is_ordered="False"
+            prompt="Find the titles of all movies directed by Steven Spielberg."
+            weight="0"
+        />
+        <sql_grader
+            dataset="rating"
+            display_name="Q2"
+            answer_query="SELECT title FROM Movie where Director = 'Steven Spielberg';"
```

Comment:
TODO: My original plan was to implement all of the real course problems inside the scenario :) The prompts here are the actual real prompts from the course; the answers just need updated.